webapp/pages/profile/_id/_slug.vue

```diff
@@ -66,16 +68,23 @@
           </ds-flex>
           <ds-space margin="small">
             <template v-if="!myProfile">
+              <base-button
+                v-if="user.isBlocked"
+                @click="unblockUser(user)"
+                class="unblock-user-button"
```

Comment:
It's probably hard to target the `.base-button` class directly since we don't have a direct (or close) parent with a unique class name? I'd prefer something like `.profile > .base-button` but that would be a bigger refactoring...